Update doc for Preference widget and advance configuration (#5681)
mendix/docs: content/en/docs/partners/siemens/3d-viewer/_index.md

### 5.9 Setting Preferences {#set-preferences}

> You can change the settings of the Preferences widget to customize the behavior of the model and setup to suit your needs.

Reviewer: Is the widget called "Preference" or "Preferences"? The image below says "Preference". 
![image](https://user-images.githubusercontent.com/35492184/221836490-c28ce167-0cf0-4db8-a8b6-f03a88004fdd.png)
```suggestion
You can change the settings of the Preference widget to customize the behavior of the model and set it up to suit your needs.
```

{{< figure src="/attachments/partners/siemens/3d-viewer/preferences-general.png" alt="preferences-general" >}}

> Click the "Preference" button to display the dialog. You can set your filter as a preference in any combination you like, and when you click the **OK** button, the filter will take effect the next time you open a file; the **Reset** button clears all settings. This method can only set the preference when your application is running, but of course we also provide a more advanced method - before the application starts - to set your preference, for more details you can refer to [Advanced Configuration](#advance_configuration).

Reviewer: ```suggestion
Click **Preference** to display the dialog. You can set your filter as a preference in any combination you like, and when you click the **OK** button, the filter will take effect the next time you open a file; the **Reset** button clears all settings. This method can only set the preference when your application is running, but of course we also provide a more advanced method - before the application starts - to set your preference, for more details you can refer to [Advanced Configuration](#advance_configuration).
```

    }
    ```

### 6.5 Preference Options

Here is a sample Preference configuration:

```json
{
    "pmiPreference": {
        "ignoredPMI": ["DIMENSION","NOTE"], 
        "ignoreModelView": true, 
        "ignoreDesignGroup": true, 
        "onlyShowAssemblyLevelPMI": false
    }
}
```

#### 6.5.1 ignoredPMI

Defines the PMI entities you do not want to display.

* Type – array

> * Default value - *[ ]*

Reviewer: This gets interpreted wrongly by markdown.
```suggestion
* Default value - *`[ ]`*
```

Available enumeration values:

```js

const ignoredPMI = [
    "BALLOON_NOTE",
    "CALLOUT_DIMENSION_TYPE",
    "CENTRELINE",
    "CHAMFER_DIMENSION_TYPE",
    "COORDINATE_SYSTEM",
    "DATUM_TARGET",
    "DIMENSION",
    "FEATURE_CONTROL_FRAME",
    "FEATURE_DATUM_TYPE",
    "FEATURE_DISCRETE_JOIN_TYPE",
    "LINE_WELD",
    "MATERIAL_SPECIFICATION",
    "MEASUREMENT_LOCATOR",
    "MEASUREMENT_POINT",
    "NOTE",
    "ORGANIZATION",
    "PART_SPECIFICATION",
    "PMI_TABLE_TYPE",
    "PROCESS_SPECIFICATION",
    "REFERENCE_AXIS",
    "REFERENCE_GEOMETRY",
    "REFERENCE_PLANE",
    "REFERENCE_POINT",
    "SECTION",
    "SPOT_WELD",

>     "SURFACE_FINISH",

Reviewer: This looks like invalid JSON - shouldn't have a comma after the last item in an array?

Author: Yes, it is a mistake from my side
